ibm_watson/assistant_v2.py

```diff
@@ -4800,12 +4485,6 @@ def __init__(self,
     def from_dict(cls, _dict: Dict) -> 'SearchResultMetadata':
         """Initialize a SearchResultMetadata object from a json dictionary."""
         args = {}
-        valid_keys = ['confidence', 'score']
```

Comment:
is this suppose to be removed?

Reply:
looking at it

Reply:
good catch, but seems like it was an additional thing we were doing that has been removed now, since `args` has been returned and not all the keys are processed:

![image](https://user-images.githubusercontent.com/16810846/101810163-71bc7f80-3ae6-11eb-9c49-af6eb4d0ed2c.png)

Reply:
oh okay I see